src/ResourceManager/Network/Commands.Network/help/New-AzureRmDdosProtectionPlan.md

```diff
@@ -0,0 +1,220 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version:
```

Comment:
@murilogr same comments in this file about build failures and output for the examples